app/views/second_level_browse_page/new_show.html.erb

```diff
@@ -11,48 +11,24 @@
 <% end %>
 
 <% content_for :breadcrumbs do %>
-  <div class="topic-page__breadcrumb-wrapper">
-    <div class="govuk-width-container">
-      <div class="govuk-grid-row">
-        <div class="govuk-grid-column-full">
-          <%= render "govuk_publishing_components/components/breadcrumbs", {
-            breadcrumbs: [
-              {
-                title: "Home",
-                url: "/"
-              },
-              {
-                title: page.active_top_level_browse_page.title,
```

Comment:
I may have missed a step but I can't see how this data gets passed to the new partials now?

Reply:
Huh so it does. @hannalaakso are you able to shed any light on that? Why we were using `page.active_top_level_browse_page.title`?

Reply:
I think this was on me, I was mirroring the hardcoded breadcrumbs on the new level 1 templates and assumed that we needed to explicitly pass through the links for the items. @patrickpatrickpatrick's solution is definitely more robust.